compiler/README.md

```diff
@@ -0,0 +1,70 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
```

Comment:
Will `<!-- XXX -->` be removed?

Reply:
No plan to remove this for now. This is just an internal comment wont be displayed.